products/compute/api.yaml

```diff
@@ -1980,6 +2033,160 @@ objects:
               is larger than the ring size, each host will be assigned a single
               virtual node.
               Defaults to 1024.
+      - !ruby/object:Api::Type::NestedObject
```

Comment:
It looks like this is copy-pasted from backendServices because they have the same API - just wanted to double-check that I understood this correctly.

Reply:
That's correct!